src/player.js

```diff
@@ -1156,13 +1159,13 @@ export default class Player extends FakeEventTarget {
   // </editor-fold>
 
   // <editor-fold desc="Fullscreen API">
-
   /**
+   * @param {boolean} isNativeOnly - check for native only
    * @returns {boolean} - Whether the player is in fullscreen mode.
    * @public
    */
-  isFullscreen(): boolean {
-    return this._fullscreen;
+  isFullscreen(isNativeOnly: boolean = false): boolean {
```

Comment:
I feel this overcomplicates the API.
You will find some implementations wanting this always with true and some with false and passing booleans in method calls is not so intuitive.
I think we should always return the `isFullscreen` and avoid the `isNativeFullscreen` altogether.
If someone calls the API and he needs to know what the meaning of passing `true` to it then he/she can just check the player config to know what is the mode.
If we see this is something that requires more definition we can expose a `getFullScreenMode()` that returns an ENUM value for `native` and `inBrowser` but this is just an overkill for now.